source/docs/articles/drupal/adding-yml-file.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Adding a services.yml File
+description: Learn how to add a services.yml file for Drupal 8.
+category:
+  - developing
+  - drupal
+keywords: drupal, services.yml, yml file
+---
+Drupal 8 includes the `sites/default/default.services.yml` configuration file to define services used, eliminating the need to hack core to swap functionality. To create and/or modify this file, you'll need to tweak some permissions, otherwise you will receive a **Permission denied** error.
+
+<div class="alert alert-danger">
+<h4>Warning</h4>
+It is not possible to modify these permissions with Git. Users working via Git should finish up their code changes and briefly switch to <a href="/docs/articles/sites/code/developing-directly-with-sftp-mode">SFTP Mode</a> to change the permissions.
```

Comment:
Okay, just tested this.  Pantheon *will* allow you to modify and then push an existing services.yml file while in "git" mode, even if the "default" folder is not writable (555), and even if the services.yml file is not writable (444).  If you do this, after the "git push", the "default" folder will remain unwritable (555), but the services yml will become writable by the owner (644).  So, when working in SFTP mode, it is important to make the files writable before you start, to avoid **Permission denied**.  In SFTP mode *and* git mode, it is important to make the files unwritable after you are done, to avoid security vulnerabilities.